src/AvTranscoder/file/OutputFile.cpp

```diff
@@ -42,6 +42,16 @@ IOutputStream& OutputFile::addVideoStream(const VideoCodec& videoDesc)
     stream.codec->level = videoDesc.getAVCodecContext().level;
     stream.codec->field_order = videoDesc.getAVCodecContext().field_order;
 
+    if (_formatContext.getAVOutputFormat().flags & AVFMT_GLOBALHEADER) {
+        stream.codec->flags = CODEC_FLAG_GLOBAL_HEADER;
```

Comment:
Because the _flags_ attribute could have a value before this assignement, I think it's better to do:
```
stream.codec->flags |= CODEC_FLAG_GLOBAL_HEADER;
```
Don't you think?